### PR TITLE
Readme doc

### PR DESCRIPTION
**Pull request by MichaelXXI**
I have created ... readme.md , accept :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# HELLO WORLD
+----------------
+
+## Into
```
Comment (mlila) [inline README.md]: @MichaelXXI "intro no", but "introduction yes"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+TODO explain lo que sea (Misha)
```
Comment (mlila) [inline README.md]: @MichaelXXI you should add an introduction

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Function description
+
+TODO Describe functionality of the app
```
Comment (mlila) [inline README.md]: @MichaelXXI please add an explanation here

File: README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+## Technical description 
+
+TODO Describe technical description of the app
```
Comment (mlila) [inline README.md]: @MichaelXXI you should add a technical desc

File: README.md (hunk 1 above)
Comment (MichaelXXI) [reply README.md]: @mlila No pienso hacerlo!!

File: README.md (hunk 2 above)
Comment (MichaelXXI) [reply README.md]: @mlila Lo puede hacer tu misma! :D jaja

File: README.md (hunk 3 above)
Comment (MichaelXXI) [reply README.md]: @mlila SISI ya voy!!

File: README.md (hunk 4 above)
Comment (MichaelXXI) [reply README.md]: @mlila Hoy lo hago!

**Pull request merged**